Code/Editor/TrackViewNewSequenceDialog.cpp

```diff
@@ -34,22 +35,107 @@ namespace
     };
 }
 
+class CTVNewSequenceDialogValidator : public QValidator
+{
+    public:
+        CTVNewSequenceDialogValidator(QObject* parent)
+            : QValidator(parent)
+        {
+            m_parentDialog = qobject_cast<CTVNewSequenceDialog*>(parent);
+        }
+
+        QValidator::State validate(QString& input, [[maybe_unused]] int& pos) const override
+        {
+            constexpr int MaxInputLength = 160;
+
+            if (input.isEmpty())
+            {
+                SetEnabled(false);
+                return QValidator::Acceptable;
+            }
+
+            bool isValid = false;
+            SetEnabled(true);
+            if (input.contains('/'))
```

Comment:
I wonder if we want to validate more than than, like all non alphanumerical characters (maybe there is already something for this somewhere in the codebase, like when we rename an asset ?)